modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/components/SimpleInputModal.es.js

```diff
@@ -61,6 +61,7 @@ const SimpleInputModal = ({
 			document.querySelector(`#${namespace}form`)
 		);
 
+		// eslint-disable-next-line @liferay/portal/no-global-fetch
```

Comment:
@bryceosterhaus, @kresimir-coko, this will need an adjustment to `no-global-fetch` since we explicitly look for `fetch.es` as a valid import which we won't have any longer... do you think we could work on this while this PR sits over here?

Reply:
It's definitely something that we can investigate, I'll open an issue for it.

Reply:
Awesome!